### ISSUE TITLE
Add class weighting to loss functions

### ISSUE THREAD

**Issue by seanmor5** [elixir-nx/axon#138]
See: https://keras.io/examples/structured_data/imbalanced_classification/
